src/msw/printdlg.cpp

```diff
@@ -187,6 +205,25 @@ bool wxWindowsPrintNativeData::IsOk() const
     return (m_devMode != NULL) ;
 }
 
+bool wxWindowsPrintNativeData::CopyAndSetDevMode(LPDEVMODE pDevMode)
+{
+    if (!pDevMode) {
+        return false;
+    }
```

Comment:
If we have this check here, we don't need the checks when calling this function. Or, conversely, if we have the checks when calling it, this should be a `wxCHECK_MSG( pDevMode, false, "null DEVMODE pointer" );` or something like this, i.e. the function shouldn't expect being called with null pointer if it's precondition is for the pointer to be non-null.

Reply:
Ok, fixed with my latest commit.